1.7/administration/installing/custom/system-requirements/install-docker-centos.md

```diff
@@ -16,6 +16,8 @@ In addition to the general [Docker requirements and recommendations for DC/OS][1
 
 * Format node storage as XFS. As of CentOS 7.2, "<a href="https://access.redhat.com/documentation/en-US/Red_Hat_Enterprise_Linux/7/html/7.2_Release_Notes/technology-preview-file_systems.html" target="_blank">only XFS is currently supported for use as a lower layer file system</a>".
 
+* Use `direct-lvm` as the method to access Docker backing store in production. For more information, see [Selecting a Storage Driver](https://docs.docker.com/engine/userguide/storagedriver/selectadriver/).
```

Comment:
Sounds reasonable to me, but I haven't tested Direct-LVM, don't know why we're recommending it, and don't know if it works with docker-in-docker (aka Jenkins). If eng wants to change the recommendation then they should change the docs to reflect that.